system/settings/win32.xml

```diff
@@ -29,13 +20,6 @@
     </category>
   </section>
   <section id="system">
-    <category id="videoscreen">
-      <group id="1">
-        <setting id="videoscreen.fakefullscreen">
```

Comment:
Windows does not have GL builds (any more). fakefullscreen is primarily a DX feature.